third_party/xla_client/BUILD

```diff
@@ -159,7 +158,7 @@ cc_library(
         "//tensorflow/compiler/xla:statusor",
         "//tensorflow/compiler/xla:xla_data_proto_cc",
         "//tensorflow/compiler/xla/service:hlo_proto_cc",
-        "//tensorflow/compiler/xla/service:hlo",
+        "//tensorflow/compiler/xla/hlo/ir:hlo",
```

Comment:
according to https://github.com/tensorflow/tensorflow/commit/25444a0fb323e2df1ce89136646c1fe49e3f2697

Reply:
revert this change

Reply:
keep this change